Samples/HelloWorld.NetCore/Readme.md

```diff
@@ -0,0 +1,30 @@
+# HelloWorld.NetCore
+Orleans HelloWorld sample targeting .NET Core
+
+## Building the Sample
+If you are trying to use a version of Orleans that is not yet released, first run `Build.cmd netstandard` from the root directory of the repository.
+
+Note: If you need to reinstall packages (for example, after making changes to the Orleans runtime and rebuilding the packages), just manually delete all Orleans packages from `(rootfolder)/Samples/{specific-sample}/packages/` and re-run `Build.cmd netstandard`. You might sometimes also need to clean up the NuGet cache folder.
+
+You can then compile as usual, build solution.
+
+```
+dotnet restore
+dotnet build
+dotnet publish
+```
+
+## Running the Sample
+It's important to publish the apps before running them, as Orleans loads several assemblies via reflection and they need to be colocated.
```

Comment:
Yeah, although what is odd is that I believe it required the client console app to have OrleansProviders, and that is not right in the first place, unless you are actually expecting to use stream providers or something like that. Nevertheless, I'll investigate later and separately from this PR (or someone else can investigate too if this PR is merged, as it's less friction that way)